_layouts/default.html

```diff
@@ -5,7 +5,7 @@
   <body>
 
     <div>
-      <a href="{{ site.baseurl }}">
+      <a href="{{ site.baseurl }}/">
```

Comment:
There is not problem on github but there is locally, that's why I've added it.
I'm not sure I would change it => I would need to change every links in the project